src/components/Pagination/pagination-utils.ts

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+
+export function getLinkDataCy(index: number, totalPages: number): string {
+  switch (index) {
+    case 0:
+      return 'prevLink';
+    case totalPages + 1:
+      return 'nextLink';
+    default:
+      return 'pageLink';
+  }
+}
+
+export function getLinkHref(index: number, totalPages: number): string {
+  switch (index) {
+    case 0:
+      return '#prev';
+    case totalPages + 1:
+      return '#next';
+    default:
+      return `#${index}`;
+  }
+}
+
+export function getLinkTextContent(index: number, totalPages: number): string {
+  switch (index) {
+    case 0:
+      return '«';
+    case totalPages + 1:
+      return '»';
+    default:
+      return `${index}`;
+  }
```

Comment:
The `getLinkTextContent` function follows the same pattern as the previous two functions. Consider simplifying it for consistency and readability.